### PR TITLE
Update homepage tokens msgid's

### PR DESCRIPTION
**Pull request by moollaza**
Related to internal change where I've updated these tokens accordingly.

Updates made with `sed`:
```
    find . -name "duckduckgo.po" -exec sed -i 's/msgid "Browse As Usual"/msgid "Browse as Usual"/g' {} +
```

/cc @andrey-p

### CODE REVIEW

**Comment by moollaza** [comment]
It appears this is causing tests to fail. @jbarrett is my approach here incorrect?

File: locales/zh_CN/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -1909,8 +1909,8 @@ msgctxt "showcase_donate"
 msgid "Join our $500,000 Privacy Challenge"
 msgstr "加入我们的五十万美金募捐行动"
 
-msgid "Just a few clicks to install, and you're ready to go. It's that easy."
-msgstr "几次点击即可完成安装，就这么简单。"
+msgid "Just a few clicks to install,%sand you're ready to go.%sIt's that easy."
+msgstr "几次点击即可完成安装，％s就可以开始使用。％s就这么简单。"
```
Comment (moollaza) [inline locales/zh_CN/LC_MESSAGES/duckduckgo.po]: @tagawa would you be able to verify this is correct?

I used Google translate to compare the before/after and they seem to be correct. Now that we have placeholders (`%s`) in the string, we need to have the same number of placeholders in the translation, so I've extended the translation to allow for this.

Comment (moollaza) [reply locales/zh_CN/LC_MESSAGES/duckduckgo.po]: /cc @b9a1 -- if you could signoff on this or suggest a better translation, that would be great!

Comment (tagawa) [reply locales/zh_CN/LC_MESSAGES/duckduckgo.po]: Yes, position of the placeholders and translation looks good to me.

Comment (b9a1) [reply locales/zh_CN/LC_MESSAGES/duckduckgo.po]:
```suggestion
msgstr "几次点击完成安装，%s即可开始使用。%s就这么简单。"
```
I would propose this as it’s more concise. Also note that the percent signs were ‘translated’ to fullwidth ones (U+FF05) when they shouldn’t.

Comment (tagawa) [reply locales/zh_CN/LC_MESSAGES/duckduckgo.po]: Oh, well spotted @b9a1 - thank you.

Comment (moollaza) [reply locales/zh_CN/LC_MESSAGES/duckduckgo.po]: @b9a1 thanks a lot!

**Comment by moollaza** [comment]
@andrey-p this is ready to merge when the internal changes are also merged 👍

**Pull request closed without merging**